_posts/2020-03-12-Steering-Group-executive-summary,-action-items,-and-minutes.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Steering Group executive summary, action items, and minutes
+author: Franklin Feingold
+display: true
+---
+
+# Steering Group executive summary, action items, and minutes
+
+Date: Thursday February 13, 2020
+Attended: Franklin Feingold, Melanie Ganz, Guiomar Niso, Robert Oostenveld, Kirstie Whitaker
+
+### Executive Summary
+
+We discussed the channels through which to share updates to the BIDS specification.
+An issue was opened to start crowdsourcing and collecting this information.
+We will be using this to share the release of v1.2.2 and to request the final rounds of feedback on the Genetic information extension.
+
+We discussed generating an example slide deck that  community members can  use (or pick from) when they want to present BIDS.
+
+We discussed considerations for when to open a repository under the BIDS standard organization on GitHub.
```

Comment:
One more sentence explaining the context ... e.g., starting with: "bids-standard is a GitHub org ... community members have in the past decided to open new repositories under the GH org umbrella ... now we want to clarify some rules for this ..."